modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js

```diff
@@ -0,0 +1,255 @@
+ * @property {object[]} form Form schema represented as an array of form items: Input, ItemSet, Layout, OptionSet.
+ */
+
+/**
+ * This function fetches a content.
+ *
+ * @example-ref examples/content/get.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} [params.versionId] Version Id of the content.
+ *
+ * @returns {object} The content (as JSON) fetched from the repository.
+ */
+exports.get = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetContentHandler');
+    bean.key = required(params, 'key');
+    bean.versionId = nullOrValue(params.versionId);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns a content attachments.
+ *
+ * @example-ref examples/content/getAttachments.js
+ *
+ * @param {string} key Path or id to the content.
+ *
+ * @returns {object} An object with all the attachments that belong to the content, where the key is the attachment name. Or null if the content cannot be found.
+ */
+exports.getAttachments = function (key) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentsHandler');
+    bean.key = nullOrValue(key);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns a data-stream for the specified content attachment.
+ *
+ * @example-ref examples/content/getAttachmentStream.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.name Attachment name.
+ *
+ * @returns {*} Stream of the attachment data.
+ */
+exports.getAttachmentStream = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentStreamHandler');
+    bean.key = required(params, 'key');
+    bean.name = required(params, 'name');
+    return bean.getStream();
+};
+
+/**
+ * This function returns the parent site of a content.
+ *
+ * @example-ref examples/content/getSite.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ *
+ * @returns {object} The current site as JSON.
+ */
+exports.getSite = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteHandler');
+    bean.key = nullOrValue(params.key);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
+};
+
+/**
+ * This function returns the site configuration for this app in the parent site of a content.
+ *
+ * @example-ref examples/content/getSiteConfig.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.applicationKey Application key.
+ *
+ * @returns {object} The site configuration for current application as JSON.
+ */
+exports.getSiteConfig = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteConfigHandler');
```

Comment:
Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)